README.md

```diff
@@ -4,91 +4,38 @@
 
-**Checkout the (very preliminary) [online version of the catalog](https://solidproject.solidcommunity.net/catalog/)!**
+**Checkout the [online version of the catalog](https://solidproject.solidcommunity.net/catalog/)!**
 
-**Note** This is a work in progress, please do not take any of the data presented as representative of the people, organizations, and software listed - the data has not been vetted.  We are creating forms and will make a call for editing and adding once the forms are ready.
+## Audience
 
-### Similar projects
-
-[Solid Efforts](https://github.com/elf-pavlik/solid-efforts) is a project intended to support exploration of the links between various ongoing work, co-creating technological foundations for the Solid ecosystem. After an attempt to use this repo as a common core, currently there is an ongoing exploration of interop/reuse of shapes, taxonomies and data between the projects.
+The primary audience of the Solid Resources Catalog is potential adopters of Solid, funders, new and onboarding developers, end-users, and the general public at any level of technical knowledge. 
 
 
 ## Long Term Goals
 
-While the catalog will start as a single hard-coded RDF document, the goal is to eventually support aggregation of data from other sources.  Personal, organizational, and software client WebID profiles contain data relevant to the catalog.  We can encourage all resources to create ClientID or WebID profiles, or a "README.ttl" - its own catalog entry describing itself which can be imported to aggregating catalogs.
-
-The task of cataloging resources is shared by most organizations so a long term goal of this project is to create reusable shapes which can be formalized into client-to-client specifications and to document the process of creating the catalog and using the shapes so that other organizations don't have to reinvent the wheel.
-
-## Scope
-
-In order to limit the work of updating the catalog, records are designed to cover only the basics - what type of thing is it? what is it useful for? what is its name and brief description? who made it? what other things is it related to? how can I find out more?  The catalog record is not like a full WebID or ClientID profile - it covers the basics and points to those types of documents for further details.
-
-## Repository structure & versioning
-
-This repo is for the asset data, the SKOS concepts, and the SHACL shape of the catalog.  For contributions to apps using these resources, see [Solid Catalog](https://github.com/solid-contrib/catalog).
-
-The repo will be versioned e.g. v1, v2, etc. This will support apps continuing to function with an older version while transitioning to a newer version.
-
-## CLI
```

Comment:
It seems useful to keep the CLI instructions since the scripts are still here

Reply:
Done!